src/pluginHooks.js

```diff
@@ -744,6 +744,14 @@ const REGISTERED_HOOKS = [
    */
   'beforeInitWalkontable',
 
+  /**
+   * Fired before new data is loaded (by `loadData` or `updateSettings` method) into the data source array.
+   *
+   * @event Hooks#beforeLoadData
+   * @param {Boolean} initialLoad flag that determines whether the data has been loaded during the initialization.
```

Comment:
The first param isn't `sourceData`?

Reply:
It should be fixed by https://github.com/handsontable/handsontable/pull/5945/commits/ef1be8085234d10b9c2341dd8fc96806f50e72e2.